hippomaps/stats.py

```diff
@@ -141,6 +146,58 @@ def moran_test(imgfix, imgperm, nperm=1000, metric='pearsonr', label='hipp', den
 
     return metricnull, imgperm_rand, pval, r_obs
 
+def eigenstrapping(imgfix, imgperm, nperm=1000, metric='pearsonr', label='hipp', den='0p5mm', num_modes=200):
```

Comment:
should add `resample:bool` and `permute:bool` as optional args here, and pass them to `SurfaceEigenstrapping` below